app/assets/javascripts/stores/AppStore.js

```diff
@@ -28,7 +28,7 @@ class AppStore {
     this.dataLoaded = (data) => {
       if ( data ) {
         console.log ( "websiteLoaded" , data )
-        this.title = data.title;
+        this = data;
```

Comment:
`this` in "this" case is the Store itself.  We want functions to be associated with `this` not only the data itself.  Therefore this would be an `extends` situation rather than a complete overwrite of `this`.  Ask me in person if you have questions.